module-bsp/board/rt1051/bsp/eink/bsp_eink.cpp

```diff
@@ -79,7 +79,7 @@ typedef struct _bsp_eink_driver
 namespace
 {
     using namespace drivers;
-    static lpspi_master_config_t s_eink_lpspi_master_config;
+    static lpspi_master_config_t spi_config;
     static std::shared_ptr<drivers::DriverGPIO> gpio;
```

Comment:
`static` seems to be redundant here, as all of these objects are declared in anonymous namespace, also `using namespace drivers` is not needed if I understand correctly, as references to classes are done with `drivers::` anyway

Reply:
Thanks, done ;)